lib/client/backend_client.dart

```diff
@@ -1,23 +1,51 @@
+//import 'dart:convert';
```

Comment:
Please delete all commented code.

Reply:
Done! Some to-do's have been left, with the TODO: removed